Source/UI/Controllers/CardViewController.swift

```diff
@@ -41,6 +41,8 @@ public class CardViewController: UIViewController,
 
     var topConstraint: NSLayoutConstraint?
 
+    private var suppressNextLog = false
```

Comment:
Will this be set to false for every view load or token generated ? , if so makes sense.